lib/oban/breaker.ex

```diff
@@ -45,6 +46,29 @@ defmodule Oban.Breaker do
     %{state | circuit: :enabled}
   end
 
+  @spec jitter(time :: pos_integer(), opts :: Keyword.t()) :: pos_integer()
+  def jitter(time, opts \\ []) do
+    mode = opts[:mode] || @default_jitter_mode
+    mult = opts[:mult] || @default_jitter_mult
+
+    diff = trunc(:rand.uniform() * mult * time)
+
+    case mode do
+      :add ->
```

Comment:
I'd expect pairs to be `:add / :sub` or `:inc / :dec`, rather than a mixture of the two.

```suggestion
      :inc ->
```